beginner_source/export_simple_model_to_onnx_tutorial.py

```diff
@@ -0,0 +1,150 @@
+# We can now export the model with the following code:
+
+export_output = torch.onnx.dynamo_export(net, input)
+
+# As we can see, we didn't need any code change on our model.
+# The resulting ONNX model is saved within ``torch.onnx.ExportOutput`` as a binary protobuf file.
+# The exporter uses static shapes by default, so the resulting model has static dimensions.
+# In a future tutorial we are going to explore how to leverage dynamic shapes and other advanced features.
+#
+# We can save it to disk with the following code:
+
+export_output.save("my_image_classifier.onnx")
+
+# Now that we have our model saved, we can visualize it with `Netron <https://github.com/lutzroeder/netron>`_.
+# Netron can either be installed on MacOS, Linux or Windows computers, or run directly from the browser.
+# Let's try the web version by opening the following link: https://netron.app/.
+#
+# .. image:: ../_static/img/onnx/netron_web_ui.png
+#
+# Once Netron is open, we can drag and drop our ``my_image_classifier.onnx`` file into the browser or select it after
+# clicking on `Open model` button.
+#
+# .. image:: ../_static/img/onnx/image_clossifier_onnx_modelon_netron_web_ui.png
+#
+# And that is it! We have successfully exported our PyTorch model to ONNX format and visualized it with Netron.
+#
+# The last step is executing the ONNX model with `ONNX Runtime`, but before we do that, let's install ONNX Runtime.
+# %%
+#  .. code-block:: bash
+#
+#      %%bash
+#      pip install onnxruntime
+
+# Adapt PyTorch input to ONNX format
+
+onnx_input = export_output.adapt_torch_inputs_to_onnx(input)
+print(f"Input legth: {len(onnx_input)}")
+print(f"Sample input: {onnx_input}")
+
+# in our example, the input is the same, but we can have more inputs
+# than the original PyTorch model in more complex cases.
+# Now we can execute the ONNX model with ONNX Runtime.
+
+import onnxruntime
+
+# We are using CPU as the execution provider, but ``providers=['CUDAExecutionProvider']`` enables CUDA too.
+
+ort_session = onnxruntime.InferenceSession("./my_image_classifier.onnx", providers=['CPUExecutionProvider'])
+
+# ONNX Runtime requires the input to be on CPU and using numpy Tensors,
+# so we need to convert our PyTorch input to numpy.
+
+def to_numpy(tensor):
+    return tensor.detach().cpu().numpy() if tensor.requires_grad else tensor.cpu().numpy()
+
+# ONNX Runtime also requires the input to be a dictionary with
+# the keys being the input name and the value the Numpy tensor
+
+onnxruntime_input = {k.name: to_numpy(v) for k, v in zip(ort_session.get_inputs(), onnx_input)}
+
+# Finally, we can execute the ONNX model with ONNX Runtime.
+
+onnxruntime_output = ort_session.run(None, onnxruntime_input)
+
+# The output can be a single tensor or a list of tensors, depending on the model.
+
+print(onnxruntime_output)
+
+# That is about it! We have successfully exported our PyTorch model to ONNX format,
```

Comment:
We should also add the matching part to show the usage of output adapter in case users are confused of mismatch